tests/fgnhg_test.py

```diff
@@ -9,187 +9,45 @@
-|                 |                    | 25             |
-|                 |                    | 26             |
-|                 |                    | 27             |
-|                 |                    | 28             |
-|                 |                    | 29             |
-|                 |                    | 30             |
-|                 |                    | 31             |
-+-----------------+--------------------+----------------+
+FG NHG Prefix    Next Hop            Hash buckets
+---------------  ------------------  ----------------------------
+fc:5::/128       200:200:200:200::4  0, 1, 2, 3, 4, 5, 6, 7
+fc:5::/128       200:200:200:200::5  8, 9, 10, 11, 12, 13, 14, 15
+100.50.25.12/32  200.200.200.4       0, 1, 2, 3, 4, 5, 6, 7
+100.50.25.12/32  200.200.200.5       8, 9, 10, 11, 12, 13, 14, 15
 """
 
 show_fgnhgv4_hash_view_output="""\
-+-----------------+---------------+----------------+
-| FG_NHG_PREFIX   | Next Hop      | Hash buckets   |
-+=================+===============+================+
-| 100.50.25.12/32 | 200.200.200.4 | 0              |
-|                 |               | 1              |
-|                 |               | 2              |
-|                 |               | 3              |
-|                 |               | 4              |
-|                 |               | 5              |
-|                 |               | 6              |
-|                 |               | 7              |
-|                 |               | 8              |
-|                 |               | 9              |
-|                 |               | 10             |
-|                 |               | 11             |
-|                 |               | 12             |
-|                 |               | 13             |
-|                 |               | 14             |
-|                 |               | 15             |
-+-----------------+---------------+----------------+
-| 100.50.25.12/32 | 200.200.200.5 | 16             |
-|                 |               | 17             |
-|                 |               | 18             |
-|                 |               | 19             |
-|                 |               | 20             |
-|                 |               | 21             |
-|                 |               | 22             |
-|                 |               | 23             |
-|                 |               | 24             |
-|                 |               | 25             |
-|                 |               | 26             |
-|                 |               | 27             |
-|                 |               | 28             |
-|                 |               | 29             |
-|                 |               | 30             |
-|                 |               | 31             |
-+-----------------+---------------+----------------+
+Alias     Next Hop       Hash buckets
+--------  -------------  ----------------------------
```

Comment:
Why are there 2 differing outputs for show fgnhg hash_view, one which contains an alias column and one which contains a prefix column, could we have both columns displayed perhaps?